<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>joueur</class>
 <widget class="QWidget" name="joueur">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>881</width>
    <height>797</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">image: url(aa.jpg);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="onglets">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="ajouter">
      <attribute name="title">
       <string>ajouter</string>
      </attribute>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>30</y>
         <width>28</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Nom :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>100</y>
         <width>43</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Prénom :</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="b_nom">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>30</y>
         <width>133</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="b_prenom">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>100</y>
         <width>133</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>180</y>
         <width>23</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Pays</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="b_pays">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>180</y>
         <width>133</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>240</y>
         <width>99</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Date de Naissance : </string>
       </property>
      </widget>
      <widget class="QDateEdit" name="b_date">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>240</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="bouton_ajouter">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>290</y>
         <width>91</width>
         <height>41</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">selection-color: rgb(85, 255, 127);</string>
       </property>
       <property name="text">
        <string>ajouter</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_ajout">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>360</y>
         <width>491</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
         <stylestrategy>PreferDefault</stylestrategy>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="bouto_annuler">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>310</y>
         <width>61</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>annuler</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="rechercher">
      <attribute name="title">
       <string>rechercher</string>
      </attribute>
      <widget class="QTableView" name="tableView_2">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>100</y>
         <width>521</width>
         <height>181</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-image:url(aa.png);</string>
       </property>
      </widget>
      <widget class="QComboBox" name="combo_chercher">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>30</y>
         <width>69</width>
         <height>21</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Nom</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Prénom</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Pays</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>30</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>recherche par</string>
       </property>
      </widget>
      <widget class="QPushButton" name="bouton_chercher">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>30</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>chercher</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="j_chercher">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>30</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="supprimer">
      <attribute name="title">
       <string>supprimer</string>
      </attribute>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>10</y>
         <width>161</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
         <underline>true</underline>
        </font>
       </property>
       <property name="text">
        <string>Liste des Joueurs :</string>
       </property>
      </widget>
      <widget class="QTableView" name="tableView">
       <property name="geometry">
        <rect>
         <x>35</x>
         <y>51</y>
         <width>517</width>
         <height>371</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="bouton_supprimer">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>450</y>
         <width>75</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>supprimer</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="id_supprimer">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>450</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>300</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>450</y>
         <width>31</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>id :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_rq">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>500</y>
         <width>551</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>bouto_annuler</sender>
   <signal>clicked()</signal>
   <receiver>label_ajout</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>311</x>
     <y>356</y>
    </hint>
    <hint type="destinationlabel">
     <x>303</x>
     <y>406</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bouto_annuler</sender>
   <signal>clicked()</signal>
   <receiver>b_date</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>283</x>
     <y>349</y>
    </hint>
    <hint type="destinationlabel">
     <x>283</x>
     <y>283</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bouto_annuler</sender>
   <signal>clicked()</signal>
   <receiver>b_pays</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>294</x>
     <y>347</y>
    </hint>
    <hint type="destinationlabel">
     <x>301</x>
     <y>226</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bouto_annuler</sender>
   <signal>clicked()</signal>
   <receiver>b_prenom</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>276</x>
     <y>351</y>
    </hint>
    <hint type="destinationlabel">
     <x>272</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bouto_annuler</sender>
   <signal>clicked()</signal>
   <receiver>b_nom</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>281</x>
     <y>355</y>
    </hint>
    <hint type="destinationlabel">
     <x>281</x>
     <y>72</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
